doc/releases/changelog-0.39.0.md

```diff
@@ -4,220 +4,384 @@
-  [(#6330)](https://github.com/PennyLaneAI/pennylane/pull/6330)
+* A new optimizer called `qml.MomentumQNGOptimizer` has been added. It inherits from the basic `qml.QNGOptimizer` 
+  optimizer and requires one additional hyperparameter: the momentum coefficient, :math:`0 \leq \rho < 1`, 
+  the default value being :math:`\rho=0.9`. For :math:`\rho=0`, `qml.MomentumQNGOptimizer` reduces down
+  to `qml.QNGOptimizer`.
+  [(#6240)](https://github.com/PennyLaneAI/pennylane/pull/6240)
 
-<h4>QJIT/JIT Compatibility</h4>
+<h4>Other Improvements</h4>
 
-* All PL templates are now unit tested to ensure JIT compatibility.
-  [(#6309)](https://github.com/PennyLaneAI/pennylane/pull/6309)
+* `process_density_matrix` was implemented in 5 `StateMeasurement` subclasses: `ExpVal`, `Var`, `Purity`, 
+  `MutualInformation`, and `VnEntropy`. This facilitates future support for mixed-state devices and 
+  expanded density matrix operations. Also, a fix was made in the `ProbabilityMP` class to use `qml.math.sqrt` 
+  instead of `np.sqrt`.
+  [(#6330)](https://github.com/PennyLaneAI/pennylane/pull/6330)
 
-* The `qml.FABLE` template now returns the correct value when JIT is enabled.
-  [(#6263)](https://github.com/PennyLaneAI/pennylane/pull/6263)
+* The decomposition for `qml.Qubitization` has been improved to use `qml.PrepSelPrep`.
+  [(#6182)](https://github.com/PennyLaneAI/pennylane/pull/6182)
 
-* `qml.QutritBasisStatePreparation` is now JIT compatible.
-  [(#6308)](https://github.com/PennyLaneAI/pennylane/pull/6308)
+* A `ReferenceQubit` device (shortname: `"reference.qubit"`) has been introduced for testing purposes 
+  and referencing for future plugin development.
+  [(#6181)](https://github.com/PennyLaneAI/pennylane/pull/6181)
 
-* `qml.AmplitudeAmplification` is now compatible with QJIT.
-  [(#6306)](https://github.com/PennyLaneAI/pennylane/pull/6306)
+* `qml.transforms.mitigate_with_zne` now gives a clearer error message when being applied on circuits, not devices, 
+  that include channel noise.
+  [(#6346)](https://github.com/PennyLaneAI/pennylane/pull/6346)
 
-* The quantum arithmetic templates are now QJIT compatible.
-  [(#6307)](https://github.com/PennyLaneAI/pennylane/pull/6307)
-  
-* The `qml.Qubitization` template is now QJIT compatible.
-  [(#6305)](https://github.com/PennyLaneAI/pennylane/pull/6305)
+* A new function called `get_best_diff_method` has been added to `qml.workflow`.
+  [(#6399)](https://github.com/PennyLaneAI/pennylane/pull/6399)
 
-<h4>Other Improvements</h4>
+* A new method called `construct_tape` has been added to `qml.workflow` for users to construct single 
+  tapes from a `QNode`.
+  [(#6419)](https://github.com/PennyLaneAI/pennylane/pull/6419)
+
+* Datasets are now downloaded via Dataset API.
+  [(#6126)](https://github.com/PennyLaneAI/pennylane/pull/6126)
```

Comment:
This won't mean anything to a reader :) What is the improvement here? Or is this just an internal implementation detail?

Reply:
Forgot to cc @DSGuala on this one!

Reply:
Re-posting from slack for convenience:

This update shouldn't change anything for the user. It's a change in how PennyLane in Python connects with the datasets online service that hosts the dataset files.